frontend/src/app/core/path-helper/path-helper.service.ts

```diff
@@ -216,91 +216,91 @@ export class PathHelperService {
     return `${this.staticBase}/roles`;
   }
 
-  public rolePath(id:string|number) {
+  public rolePath(id:string | number) {
     return `${this.rolesPath()}/${id}`;
   }
 
   public versionsPath() {
     return `${this.staticBase}/versions`;
   }
 
-  public versionEditPath(id:string|number) {
+  public versionEditPath(id:string | number) {
     return `${this.staticBase}/versions/${id}/edit`;
   }
 
-  public versionShowPath(id:string|number) {
+  public versionShowPath(id:string | number) {
     return `${this.staticBase}/versions/${id}`;
   }
 
   public workPackagesPath() {
     return `${this.staticBase}/work_packages`;
   }
 
-  public workPackagePath(id:string|number) {
+  public workPackagePath(id:string | number) {
     return `${this.staticBase}/work_packages/${id}`;
   }
 
-  public workPackageShortPath(id:string|number) {
+  public workPackageShortPath(id:string | number) {
     return `${this.staticBase}/wp/${id}`;
   }
 
-  public workPackageCopyPath(workPackageId:string|number) {
+  public workPackageCopyPath(workPackageId:string | number) {
     return `${this.workPackagePath(workPackageId)}/copy`;
   }
 
-  public workPackageDetailsPath(projectIdentifier:string, workPackageId:string|number, tab?:string) {
+  public workPackageDetailsPath(projectIdentifier:string, workPackageId:string | number, tab?:string) {
     if (tab) {
       return `${this.projectWorkPackagePath(projectIdentifier, workPackageId)}/details/${tab}`;
     }
 
     return `${this.projectWorkPackagesPath(projectIdentifier)}/details/${workPackageId}`;
   }
 
-  public workPackageDetailsCopyPath(projectIdentifier:string, workPackageId:string|number) {
+  public workPackageDetailsCopyPath(projectIdentifier:string, workPackageId:string | number) {
     return this.workPackageDetailsPath(projectIdentifier, workPackageId, 'copy');
   }
 
-  public workPackageReminderModalBodyPath(workPackageId:string|number) {
+  public workPackageReminderModalBodyPath(workPackageId:string | number) {
     return `${this.workPackagePath(workPackageId)}/reminders/modal_body`;
   }
 
-  public workPackageSharePath(workPackageId:string|number) {
+  public workPackageSharePath(workPackageId:string | number) {
     return `${this.workPackagePath(workPackageId)}/shares`;
   }
 
-  public workPackageHoverCardPath(workPackageId:string|number) {
+  public workPackageHoverCardPath(workPackageId:string | number) {
     return `${this.workPackagePath(workPackageId)}/hover_card`;
   }
 
-  public workPackageProgressModalPath(workPackageId:string|number) {
+  public workPackageProgressModalPath(workPackageId:string | number) {
     if (workPackageId === 'new') {
       return `${this.workPackagePath(workPackageId)}/progress/new`;
     }
 
     return `${this.workPackagePath(workPackageId)}/progress/edit`;
   }
 
-  public workPackageUpdateCounterPath(workPackageId:string|number, counter:string) {
+  public workPackageUpdateCounterPath(workPackageId:string | number, counter:string) {
```

Comment:
Your editor reformatted all those lines to use spacing between typedefs, even though our eslint/editorconfig expects no space between

Reply:
If this is in the config, why is the autofixer not applying it?

```
$ npm run lint:eslint -- --fix src/app/core/path-helper/path-helper.service.ts

> openproject-frontend@0.1.0 lint:eslint
> eslint --fix src/app/core/path-helper/path-helper.service.ts

$ git status
On branch implementation/59037-primerize-log-time-dialog
Your branch is up to date with 'origin/implementation/59037-primerize-log-time-dialog'.

nothing to commit, working tree clean
```

Reply:
manually removed those changes